File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int leng = nums.size();
        vector<int> res(2 * leng);
        for (int i = 0; i < leng; i++) {
            res[i] = res[i+leng] = nums[i];
        }
        return res;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashMap = {}
        for i in range(len(nums)):
            diff = target-nums[i]
            if diff in hashMap:
                return [hashMap[i], i]

            hashMap[num[i]] = i